Source/Core/Core/NetPlayClient.cpp

```diff
@@ -535,10 +534,7 @@ void NetPlayClient::OnChatMessage(sf::Packet& packet)
   INFO_LOG_FMT(NETPLAY, "Player {} ({}) wrote: {}", player.name, player.pid, msg);
 
   // add to gui
-  std::ostringstream ss;
-  ss << player.name << '[' << char(pid + '0') << "]: " << msg;
```

Comment:
oh god it took me a bit to understand why this was equivalent, why was the code previously like this

Reply:
This code looks really sketchy in general... is that a vector access from a value read from the network? Eww.

Reply:
Potentially worse, it's a std::map::operator[] access from a value read from the network.  I don't understand why a map is being used when the key type is a u8 with a fancy name...?